documentation/tutorial/02-reactivity/01-reactive-assignments/app-a/App.svelte

```diff
@@ -5,7 +5,11 @@
 		// event handler code goes here
 	}
 </script>
-
+<style>
```

Comment:
Hi! Thanks for this change! A few things: 

- Can you use prettier extension if you are using VSCode.
- `<style>` block is supposed to be at the bottom of the file, after the markup finishes.
- App.svelte in `app-b` folder needs to be updated with the same change as well

Reply:
Thank you for the feedback! Have made the required changes as needed. Would request you to go through it.